libs/plugins/stream-client/src/lib/core/state/clean-dangling-tasks-output-mappings.ts

```diff
@@ -0,0 +1,68 @@
+import { isEmpty, pick, fromPairs } from 'lodash';
+import { ContributionSchema } from '@flogo-web/core';
+import { Dictionary } from '@flogo-web/lib-client/core';
+import { isMapperActivity } from '@flogo-web/plugins/flow-core';
```

Comment:
We should not be loading the logic from `flow-core` as you are mixing the plugin logic internally